File: 백준/Silver/1427. 소트인사이드/소트인사이드.py
#  **************************************************************************  #
#                                                                              #
#                                                       :::    :::    :::      #
#    Problem Number: 1427                              :+:    :+:      :+:     #
#                                                     +:+    +:+        +:+    #
#    By: hrkim2001 <boj.kr/u/hrkim2001>              +#+    +#+          +#+   #
#                                                   +#+      +#+        +#+    #
#    https://boj.kr/1427                           #+#        #+#      #+#     #
#    Solved: 2025/09/09 23:37:55 by hrkim2001     ###          ###   ##.kr     #
#                                                                              #
#  **************************************************************************  #
A = list(input())
len_A = len(A)

for i in range(len_A - 1):
    max = i
    for j in range(i + 1, len_A):
        if A[max] < A[j]:
            max = j
    A[i], A[max] = A[max], A[i]

for i in range(len_A):
    print(A[i], end='')

File: 백준/Silver/1427. 소트인사이드/소트인사이드.java
/* ************************************************************************** */
/*                                                                            */
/*                                                      :::    :::    :::     */
/*   Problem Number: 1427                              :+:    :+:      :+:    */
/*                                                    +:+    +:+        +:+   */
/*   By: hrkim2001 <boj.kr/u/hrkim2001>              +#+    +#+          +#+  */
/*                                                  +#+      +#+        +#+   */
/*   https://boj.kr/1427                           #+#        #+#      #+#    */
/*   Solved: 2026/01/04 23:33:33 by hrkim2001     ###          ###   ##.kr    */
/*                                                                            */
/* ************************************************************************** */
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        char[] arr = sc.next().toCharArray();

        Arrays.sort(arr);
        for (int i = arr.length - 1; i >= 0; i--) {
            System.out.print(arr[i]);
        }
    }
}
